<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plot</class>
 <widget class="QDialog" name="plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>482</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>482</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>482</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmallPAD</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>f(x)=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_fx"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>xmin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_xmin"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>xmax</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="le_xmax"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ymin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="le_ymin"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>ymax</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="le_ymax"/>
     </item>
     <item row="1" column="0" colspan="8">
      <widget class="QCustomPlot" name="view" native="true">
       <property name="minimumSize">
        <size>
         <width>658</width>
         <height>331</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>658</width>
         <height>331</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_plot">
       <property name="text">
        <string>Отрисовка</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pb_export">
       <property name="text">
        <string>Экспорт...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
